dist/ng-context-menu.js

```diff
@@ -107,16 +107,15 @@
             });
 
             function handleKeyUpEvent(event) {
-              if (!$scope.disabled() && opened && event.keyCode === 27) {
+              if (opened && event.keyCode === 27) {
                 $scope.$apply(function() {
                   close(ContextMenuService.menuElement);
                 });
               }
             }
 
             function handleClickEvent(event) {
-              if (!$scope.disabled() &&
-                opened &&
+              if (opened &&
```

Comment:
Does this work ok in Firefox? The comment a few lines further down suggests it does something weird with the events where we get a click event and a `contextmenu` event.

Reply:
I've tested this in Firefox 80 and it works fine.